capsule-sync-check.sh

```diff
@@ -32,6 +39,13 @@ then
     sleep 300
     count_capsule_syncs_in_progress
     warn "${_CAPSULE_SYNC_COUNT} sync jobs to capsules still running"
+    ((WAIT+=300))
+    if [[ ${WAIT} -gt ${MAX_SYNC_WAIT} ]]
```

Comment:
This is the wrong place to abort.

You've already waited another 5 minutes, and called `count_capsule_syncs_in_progress` again, but never evaluated the result. I'd put the abort *before* the `inform/sleep` at the start of the `while`

Reply:
full ACK
changed in 6fc3f105a16d26c90530229bcfa32697589aeec3
will give it a test run now, just to be clean.